ParabolInc/parabol
# Disable refetch queries on new version bump

#5548

See discussion starting from here: https://github.com/ParabolInc/parabol/issues/5548#issuecomment-984867643

How to test:

**1.**

- `yarn build && yarn start`
- Open any page
- Shut down the server
- See 'You are offline' snackbar
- Run `yarn build && yarn start` again
- See queries are refetched after reconnect, the offline message is gone (you probably would see the page goes blank for a second)

**2.**

- `yarn build && yarn start`
- Open any page
- Shut down the server
- See 'You are offline' snackbar
- Edit the version in `package.json` in the root folder
- Run `yarn build && yarn start`
- See offline message is gone
- See that there was no refetch
